pytensor/link/c/cmodule.py

```diff
@@ -1961,14 +1961,14 @@ def _try_flags(
             return False
 
         code = (
-            """
+            f"""
         {preamble}
         int main(int argc, char** argv)
         {{
             {body}
             return 0;
         }}
-        """.format(**locals())
+        """
```

Comment:
Shouldn't `**locals` expand to a dictionary of all function parameters, not just `preamble`?

Reply:
`locals()` is a dictionary of all the variables in scope, so it expands here to `preamble` and `body`, but also to others. However, `format` doesn't use its unnecessary arguments.
``` python
assert "{abc}".format(abc="X", xyz="Y") == "X"
```